honeybee_radiance_command/options/gendaylit.py

```diff
@@ -0,0 +1,195 @@
+# coding: utf-8
+from .optionbase import (
+    OptionCollection,
+    BoolOption,
+    NumericOption,
+    IntegerOption,
+    TupleOption,
+)
+import warnings
+
+
+class GendaylitOptions(OptionCollection):
+    """Gendaylit command options.
+
+    Also see: https://floyd.lbl.gov/radiance/gendaylit.1.html
+    """
+
+    __slots__ = (
+        '_P',
+        '_W',
+        '_L',
+        '_O',
+        '_s',
+        '_g',
+        '_a',
+        '_o',
+        '_m',
+        '_ang'
+        )
+
+    def __init__(self):
+        """Gendaylit command options."""
+
+        OptionCollection.__init__(self)
+
+        self._P = TupleOption("P", "Espilon data a.k.a Perez parameters", length=2)
+        self._W = TupleOption("W", "Direct normal irradiance and diffuse"
+                              " horizontal irradiance", length=2, numtype=int)
+        self._L = TupleOption("L", "Direct normal illuminance and"
+                              " diffuse horizontal illuminance", length=2, numtype=int)
+        self._O = IntegerOption("O", "Output", min_value=0, max_value=2)
+        self._s = BoolOption("s", "Generate source description of the sun")
+        self._g = NumericOption("g", "Average ground reflectance")
+        self._a = NumericOption("a", "Site latitude", min_value=-90.0, max_value=90.0)
+        self._o = NumericOption("o", "Site longitude", min_value=-180.0, max_value=180.0)
+        self._m = NumericOption("m", "Standard meridian", min_value=-18.0,
+                                max_value=18.0)
+        self._ang = TupleOption("ang", "Altitude & azimuth", length=2, numtype=float)
+        self._on_setattr_check = True
+
+    def _on_setattr(self):
+        """This method executes after setting each new attribute.
+        Use this method to add checks that are necessary for OptionCollection.
+        """
+        if self._ang.is_set and (
+                self._a.is_set or self._o.is_set or self._m.is_set):
+            warnings.warn(
+                'Options -a, -o and -m do not apply when -ang is set.'
+            )
```

Comment:
I think this should be considered an error and not a warning. Is there a reason that you want to still allow this combination?